support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouSupportReminder.jsx

```diff
@@ -51,15 +51,22 @@ const privacyTextLink = css`
   color: ${neutral[20]};
 `;
 
-interface ReminderSignup {
-  reminderPeriod: string;
-  reminderOption: string;
+interface RecurringReminderChoice {
+  type: 'RECURRING';
+  signup: {
+    reminderFrequencyMonths: number;
+  };
+  label: string;
 }
-
-interface ReminderChoice {
-  signup: ReminderSignup;
+interface OneOffReminderChoice {
+  type: 'ONE_OFF';
+  signup: {
+    reminderPeriod: string;
+    reminderOption: string;
+  };
   label: string;
 }
+type ReminderChoice = RecurringReminderChoice | OneOffReminderChoice;
```

Comment:
I initially tried to modify the existing `ReminderChoice` model with `signup: RecurringReminderSignup | OneOffReminderSignup`.
But flow gets confused, even if they have a `type` field